### PR TITLE
Admin Portal Navigation

### PR DESCRIPTION
**Pull request by rluberto**
In this PR I updated the admin portal and added the new navigation bar. Let me know if you want me to change anything.

image

image

image

### CODE REVIEW

**Review by tbatk — APPROVED** [review]
Looks awesome! Do you think you could have some sort of component that dims the background when you open the menu on mobile? It could be a z-index higher than the page, but not higher than the menu. Clicking it could close the nav menu. That way users don't click on the buttons in the background when the nav menu is open.

File: components/Portal/Navigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                                Resources
+                            </a>
+                            <a className={style.navLowerBtn} href="/faq">
+                                FAQ
+                            </a>
+                        </div>
+                    </div>
+                    <div className={style.navLowerText}>
+                        A Hack4Impact Chapter
+                    </div>
```
Comment (VishalAiely) [inline components/Portal/Navigation/index.tsx]: We forgot to remove this section from the design but this part can be erased

**Review by VishalAiely — APPROVED** [review]
The dashboard looks really good and follows the design really well

**Review by kemalfidan — APPROVED** [review]

**Comment by rluberto** [comment]
I added the overlay for the mobile menu @trevormangrum and I replaced the bottom text with their slogan @VishalAiely . Let me know if you want to keep their slogan at the bottom or remove that text completely. Also clicking on the blacked-out area will close the menu.

image

**Comment by VishalAiely** [comment]
I think the statement at the bottom looks good @rluberto

**Pull request merged**